cdap-ui/server/express.js

```diff
@@ -121,14 +121,21 @@ function extractUITheme(cdapConfig, uiThemePath) {
     }
   }
   // Relative path
+  
   {
     let themePath;
+    
     try {
       // __dirname will always be <entire-cdap-home>/ui/server.
       // So the uiThemePath should be relative to the ui folder
       // Ideally this will be of the form 'server/config/themes/default|light.json
       // For development since we are starting node from the ui folder we need to drop
       // the 'server' from the beginning of the path (config/themes/default|light.json)
+
+      if (uiThemePath.slice(0, 6) === 'server') {
```

Comment:
nit: Can we use `uiThemePath.startsWith` function? Helps us avoid slicing and tracking indexes. [MDN reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) for startsWith